contracts/.solhint.json

```diff
@@ -3,11 +3,10 @@
   "plugins": ["prettier"],
   "rules": {
     "code-complexity": ["error", 7],
-    "compiler-version": ["error", "^0.6.12"],
```

Comment:
It seems the received funds are not sorted properly. You can see the most recently received funds are at the bottom, but they should be at the top

<img width="877" alt="image" src="https://user-images.githubusercontent.com/17163988/159098931-db84e14e-a19e-4919-93e0-92e4cd5fbb02.png">

Reply:
So weird. I saw this a bit testing last week, but can't reproduce it today. My guess is it's a race-condition issue of some kind, and related to web workers. I think make an issue for this one and we can tackle it at some point later.

Reply:
Tracking in https://github.com/ScopeLift/umbra-protocol/issues/316